py/bitbox02/bitbox02.py

```diff
@@ -546,6 +548,8 @@ def restore_from_mnemonic(self) -> bool:
         request = hww.Request()
         # pylint: disable=no-member
         request.restore_from_mnemonic.CopyFrom(hww.RestoreFromMnemonicRequest())
```

Comment:
i think this line is not needed anymore since you set the vars below.

Alternatively, you can set the fields as args to `hww.RestoreFromMnemonicRequest(timestamp=...)`.

Reply:
ack